srcpkgs/micropython/patches/ppc64.patch

```diff
@@ -1,20 +0,0 @@
---- py/emitglue.c	2018-12-20 16:15:21.717553255 +0100
```

Comment:
did you check that this patch really is no longer needed? Unless upstream integrated the changes, it might just need refreshing to apply to the new version.

Reply:
The code changed a lot in https://github.com/micropython/micropython/commit/6810f2c134f9329e9dc18f4e0d3a1936ca6d8011. In 1.9.4 similar code is in https://github.com/micropython/micropython/blob/v1.9.4/py/persistentcode.c#L380 and `defined(__unix__)` applies to all architectures, so if I understand the code correctly the patch isn't necessary anymore.